beets/dbcore/db.py

```diff
@@ -71,10 +84,10 @@ def __getitem__(self, key):
         else:
             raise KeyError(key)
 
-    def __iter__(self):
+    def __iter__(self) -> Iterable:
```

Comment:
Couldn't this be made more specific as `Iterable[str]`?

Reply:
Fixed in e57d66e2